### PR TITLE
CVE-2019-14904

### PR DESCRIPTION
**Pull request by pedrohc**

### CODE REVIEW

File: 2019/14xxx/CVE-2019-14904.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                    {
+                        "lang": "eng",
+                        "value": "CWE-78"
+                    }
+                ]
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14904.json]: Please add corresponding upstream pull request as reference:

 https://github.com/ansible/ansible/pull/65686

**Comment by iamleot** [comment]
@cve-team can we please wait for possible replies/changes before merging them?

Thanks!

**Comment by guitos** [comment]
@iamleot, the CVE-2019-14904 page was update to include the upstream reference mentioned in previous comments, see https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-14904.

**Comment by iamleot** [comment]
Thank you @guitos!

**Pull request merged**